### DISCUSSION TITLE
`productDetail.image` is inappropriately resized when browser window is resized

### DISCUSSION THREAD

**Discussion by dorianj-teradata** [microsoft/Microsoft-Fabric-workload-development-sample#50] [Help] [answered]
I've se the background image for my workload's metadata page. It appears correctly, however I was surprised to see the aspect ratio get messed up when the browser window is resized horizontally. I expect the image to get clipped (either on left or right, not picky -- but preferably one of those and not both).

**Comment by omgharra** [comment]
Can you please attach a screenshot?

**Comment by dorianj-teradata** [comment]
yeah -- here's window resized small
<img width="643" alt="Screenshot 2024-07-31 at 9 07 46 AM" src="https://github.com/user-attachments/assets/5dec6d1c-6707-469f-a933-cc1943ba6f4b">

and large 
<img width="2154" alt="Screenshot 2024-07-31 at 9 08 12 AM" src="https://github.com/user-attachments/assets/e6b86cbf-8ba8-4a70-b784-0c3440dc3c4b">

i expect the hero background image to not get squished or stretched

**Comment by omgharra** [comment]
Checking...

**Comment by omgharra** [comment]
Working on a fix - will provide ETA when I have

**Comment by omgharra** [comment, marked as answer]
@dorianj-teradata Fix is expected to be deployed to production beginning of October, thank you for reporting.

**Comment by dorianj-teradata** [comment]
Sweet! Thanks so much for looking at this, our Marketing folks will be happy to hear!

**Comment by omgharra** [comment]
Sure, happy to help.
Can you please mark it as answered?
